metrics/robinhood.py

```
@@ -0,0 +1,127 @@
+
+
+class HTTPEthCallLatencyMetric(HttpCallLatencyMetricBase):
+    """Collects response time for eth_call simulation."""
+
+    @property
+    def method(self) -> str:
+        """Return the RPC method name."""
+        return "eth_call"
+
+    @staticmethod
+    def get_params_from_state(state_data: dict) -> list:
```

Ruff (0.15.21)

[warning] Unused static method argument: `state_data`

(ARG004)